app/webpacker/packs/application.js

```diff
@@ -2,6 +2,7 @@ import '../scripts/govuk_assets_import'
 import '../styles/application.scss'
 import '../scripts/components'
 import '../scripts/global/nationality_select'
+import '../scripts/global/disable-autocomplete'
```

Comment:
what about `disable-browser-autofill`?

I know the actual attribute set is autocomplete, but people confuse these with other sots of autocomplete - and what we're actually trying to target is browser autofill.